001_github_actions_importing_workflows/README.md

```diff
@@ -54,7 +54,7 @@ Use `cue` to import your YAML workflow files:
 
 :computer: `terminal`
 ```sh
-cue import ./.github/workflows/ --with-context -p github -f -l workflows: \ -l 'strings.TrimSuffix(path.Base(filename),path.Ext(filename))'
+cue import ./.github/workflows/ --with-context -p github -f -l workflows: -l 'strings.TrimSuffix(path.Base(filename),path.Ext(filename))'
```

Comment:
what was this escaped space meant to do? why did it even work before?

Reply:
It was meant to be an escaped newline, so that the reader had visibility of the whole command on the rendered page, but that was removed during initial review. the backslash persisted, annoyingly :-\